### DISCUSSION TITLE
Lesson 5: Private Keys/wallets error in Brownie

### DISCUSSION THREAD

**Discussion by ArcusConsulting** [smartcontractkit/full-blockchain-solidity-course-py#224] [Q&A] [answered]
Everything  in Lesson 5 has been working until 4:42.

This is my brownie_config.yaml file:
![image](https://user-images.githubusercontent.com/91853555/138014095-d9d48bad-ab4d-47c7-a72c-04f3d446a68a.png)

My .env file:
![image](https://user-images.githubusercontent.com/91853555/138014191-3eeaa13c-24a4-446a-9f37-47cda84e730f.png)

And my deploy.py file:
![image](https://user-images.githubusercontent.com/91853555/138014245-b9177c94-0953-498b-a711-4d8b93807c2c.png)

And my output:
![image](https://user-images.githubusercontent.com/91853555/138014347-90f34041-f087-4366-b7aa-76efc3355742.png)

Don't know what the error means; it's referencing some brownie files that I don't even have open...

**Comment by ArcusConsulting** [comment]
And my files are organized as such:

![image](https://user-images.githubusercontent.com/91853555/138014498-13f36f5d-78d1-4fc8-99f0-dcd79648d261.png)

**Reply by PatrickAlphaC** [reply]
@ArcusConsulting PLEASE pull any funds you have out of that private key. If its just testnet funds, it's ok. But if there are real funds in there, you will get all money stolen. Please don't share private keys.

**Reply by ArcusConsulting** [reply]
Yea it's just testnet funds.

**Comment by ArcusConsulting** [comment]
It doesn't like my "wallets" in the deploy.py file but I have no idea why.

**Reply by PatrickAlphaC** [reply]
You likely just have some weird characters. Can you copy-paste from the example and see if it works? 
https://github.com/PatrickAlphaC/brownie_simple_storage/blob/main/scripts/deploy.py

**Reply by ArcusConsulting** [reply]
Will try tomorrow and let you know.

**Reply by ArcusConsulting** [reply]
It was the format of the yaml file name... changed it to brownie-config instead of brownie_config and I'm good to go!

**Comment by cromewar** [comment, marked as answer]
I see a lot of people using `brownie_config.yaml` instead of `brownie-config.yaml` remember you must use `-` and if you notice, you have it with `_` that's why brownie is not able to find the configuration file, give it a try ;)

**Reply by ArcusConsulting** [reply]
Didn't think that would make a difference since it's just the file's name. Will definitely change it today and give an update after work, thank you!

**Reply by cromewar** [reply]
Hope it works, best regards.

**Reply by ArcusConsulting** [reply]
Worked! That was the issue. Thank you!!

**Reply by CedricVo91** [reply]
I have the same problem, however, my yaml file is called correctly...

**Reply by cromewar** [reply]
Look for any indentation errors @CedricVo91 or any typos.
